README.md

```diff
@@ -17,8 +17,35 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
         def application do
           [applications: [:cog_api]]
         end
+## Stucture
 
-## Test
+There are two main areas of CogApi.
+
+### CogApi.HTTP.Client
+This is the primary client that is recommended for usage. This is being tested
+as it is updated.
+
+Alongside this, there is a `CogApi.Fake.Client`. This is designed for testing
+and local development purposes. The API for `Fake.Client` and `HTTP.Client` are
```

Comment:
Worth mentioning it will never actually hit Cog? It may not be obvious to some. `This is designed for testing and local development purposes, and will never hit real Cog instance.`